rest-api-spec/src/main/resources/rest-api-spec/test/nodes.stats/30_discovery.yml

```diff
@@ -26,4 +28,6 @@
   - is_false: nodes.$master.name
   - is_false: nodes.$master.jvm
   - is_true:  nodes.$master.discovery
+  - is_true:  nodes.$master.discovery.cluster_state_queue
+  - is_true:  nodes.$master.discovery.published_cluster_states_received
```

Comment:
can we also check the sub elements?

Reply:
So `is_true` fails on the subfields because 0 is falsey. I'm leaning towards saying `gte: 0` as a proxy for "is present and numeric".